AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -107,56 +107,69 @@ struct SingleMeasurementView: View {
     let measurementPresentationStyle: MeasurementPresentationStyle
     
     var body: some View {
-        
         VStack(spacing: 3) {
             Text(showStreamName())
                 .font(Font.system(size: 13))
-            if measurementPresentationStyle == .showValues {
-                Button(action: {
-                    selectedStream = stream
-                }, label: {
-                    if let value = value,
-                       let threshold = threshold {
-                        HStack(spacing: 3) {
-                            MeasurementDotView(value: value,
-                                               thresholds: threshold)
-                            Text("\(Int(value))")
-                                .font(Font.moderate(size: 14, weight: .regular))
-                        }
-                        
-                    }
-                })
-                .buttonStyle(AirCastingStyling.BorderedButtonStyle(isSelected: selectedStream == stream,
-                                                                   thresholdColor: colorBorder(stream: stream)))
+            if measurementPresentationStyle == .showValues,
+               let value = value,
+               let threshold = threshold {
+                _SingleMeasurementButton(stream: stream,
+                                         value: value,
+                                         selectedStream: $selectedStream,
+                                         threshold: threshold)
             }
         }
     }
     
     func showStreamName() -> String {
         guard let streamName = stream.sensorName else { return "" }
         if streamName == Constants.SensorName.microphone {
-              return "db"
+            return "db"
         } else {
             return streamName
                 .drop { $0 != "-" }
                 .replacingOccurrences(of: "-", with: "")
         }
     }
-    
-    func colorBorder(stream: MeasurementStreamEntity) -> Color {
-        guard let value = value else { return .white }
-        guard let threshold = threshold else { return .white }
-        switch Int32(value) {
-        case threshold.thresholdVeryLow..<threshold.thresholdLow:
-            return .aircastingGreen
-        case threshold.thresholdLow..<threshold.thresholdMedium:
-            return .aircastingYellow
-        case threshold.thresholdMedium..<threshold.thresholdHigh:
-            return .aircastingOrange
-        case threshold.thresholdHigh..<threshold.thresholdVeryHigh:
-            return .aircastingRed
-        default:
-            return .white
+
+    struct _SingleMeasurementButton: View {
```

Comment:
not sure why there is an underscore in the name of this struct. It also might be beneficial to put this in a separate file.

Reply:
Not sure if you want to move `SingleMeasurementView` or `_SingleMeasurementButton` to separate file. I think it makes more sense to move the whole `SingleMeasurementView`. I can also make `_SingleMeasurementButton` a separate struct (it's nested in `SingleMeasurementView`). 
And the underscore. It's just a mark that this struct is private. Some apple frameworks have the underscore in names to mark is as private so it was kind of tribute to them.🙃 But I'll remove it if it's confusing.